### PR TITLE
60 build creatematch component

### PR DESCRIPTION
**Pull request by alaaahmade**
60 build createMatch component #60

### CODE REVIEW

File: server/services/matches.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ const createMatchService = async (
   const newStartTime = startDate;
   const newEndTime = endDate;
 
+  console.log(body);
```
Comment (ahmedirheem) [inline server/services/matches.ts:15]: remove log

**Review by ahmedirheem — APPROVED** [review]
Just there is one comment

File: client/src/components/calender/Calender.tsx [hunk 2]
```diff
@@ -41,26 +45,65 @@ const Calendar = (): ReactElement => {
     modifiedArg.el.style.cursor = 'pointer';
     modifiedArg.el.style.backgroundColor = '#2CB674';
   };
+
+  const AddNewEvent = async (newEvent: newEventInterface) => {
+    await setDate(
```
Comment (Saeed99Madi) [inline client/src/components/calender/Calender.tsx:50]: serDate function does not return a `promise`, so there is no need to use `await`

**Review by ahmedirheem — APPROVED** [review]

**Review by Saeed99Madi — CHANGES_REQUESTED** [review]
Well Done Alaa, 
Great Effort, 
@alaaahmade

**Review by ahmedirheem — APPROVED** [review]

**Review by Saeed99Madi — APPROVED** [review]

**Review by Emanalabsi — APPROVED** [review]

**Review by Heba-Adwan — APPROVED** [review]

**Pull request merged**